README.md

```diff
@@ -1 +1,40 @@
-# LeaderBoard-project
\ No newline at end of file
+# LeaderBoard-project
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the simple to do list: using webpack that shows of my accomplishments as a software developer.In this project  we learn how to setup the webpack and nodemodules. This project do the to-do-list add , remove and edit function with local storage.
+
+
+
+## Built With
+
+- HTML
+- Css
+- Javascript
+- Es6
+- webpack
+- gitflow
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] [OPTIONAL] Kindly consider updating your readme file with a detailed information on how to set up and start the project, It would be very beneficial to you in the near future since you may not know it off-head. :wink: 
- [ ] [OPTIONAL] Please update your readme file with the link to the license to make your project looks more proffessional